packages/frontend/src/__tests__/blockButton.js

```diff
@@ -1,8 +1,17 @@
 import { screen, render, waitFor } from '@testing-library/react'
+import UserContext from '../context/User'
 import userEvent from '@testing-library/user-event'
 import BlockButton from '../components/postBlock/blockButton'
 
-test('should render BlockButton props correctly', () => {
+const renderBlockButton = (userData, type, count, data) => {
+    return render(
+        <UserContext.Provider value={userData}>
+            <BlockButton type={type} count={count} data={data} />
+        </UserContext.Provider>
+    )
+}
+
+test('should render BlockButton props correctly with comment button', () => {
     // Mocked props
     const type = 'comment'
     const count = 1
```

Comment:
Can we change this count variable to something more unique? I guess it’s unlikely that the number 1 is hanging around in the document, but it’s less likely that 274728 is in the document.